spec/services/transcript_service_spec.rb

```diff
@@ -113,4 +113,31 @@
       end
     end
   end
+
+  describe ".find" do
```

Comment:
should this be find_by_uid ? (or just find)

Reply:
@craighawkes, thanks , my mistake, I changed the name later and forgot to update the spec, will fix that now..